strings.asm

```diff
@@ -15,13 +15,13 @@
 ; even when some are missing because they were removed in the
 ; platform file.
 ix := 0
-str_ok             = ix
+str_ok             = ix         ; unused?
```

Comment:
i tried make with `-Wunused` and some of these strings are flagged as unused - perhaps could remove?